### PR TITLE
Enable store-constants-to-file by default

### PR DESCRIPTION
**Pull request by tungld**
Many users of onnx-mlir are not aware of the option `--store-constants-to-file` that would be used for big models. Thus, this patch enables store-constants-to-file by default so that constants will be automatically stored on file whenever the total size of the constants is >= 1.5GB.

### CODE REVIEW

File: src/Compiler/CompilerOptions.cpp [hunk 1]
```diff
@@ -337,13 +337,16 @@ static llvm::cl::opt<bool, true> storeConstantsToFileOpt(
...
-        "into the model.so. The binary file is in the same folder as the "
-        "model.so and has the same name as the model with the extension of "
-        ".constants.bin. For inference, model.constants.bin must be at the "
-        "same folder as the inference program. If model.constants.bin is at "
-        "another folder, use the environment variable OM_CONSTANT_PATH to set "
-        "the constant folder. Windows will be supported soon."),
-    llvm::cl::location(storeConstantsToFile), llvm::cl::init(false),
+        "into the model.so when the sizes of constants satisfy thresholds "
+        "defined by options constants-to-file-single-threshold and "
+        "constants-to-file-total-threshold. The binary file is in the same "
```
Comment (AlexandreEichenberger) [inline src/Compiler/CompilerOptions.cpp:342]: Nit: I would add "When using the model, "... to specify when that co-location and/or env variable must be defined.

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

File: src/Compiler/CompilerOptions.cpp (hunk 1 above)
Comment (tungld) [reply src/Compiler/CompilerOptions.cpp]: I added this `When using this option, two other options constants-to-file-single-threshold and constants-to-file-total-threshold can be used to finetune the amount of constants stored on the file.`. Hope it's fine. Thanks!

**Comment by tungld** [comment]
@jenkins-droid test this please

**Comment by tungld** [comment]
@jenkins-droid test this please

**Pull request merged**